django/cantusdb_project/main_app/tests/test_functions.py

```diff
@@ -56,57 +51,33 @@ def test_concordances_structure(self):
         with self.subTest(test="Ensure no unexpected keys present"):
             self.assertEqual(len(concordance_keys), len(expected_keys))
 
-    def test_number_of_concordances_returned(self):
-        cantus_ids: tuple[tuple[str, int]] = (
-            ("000002", 2),
-            ("000003", 3),
-            ("000005", 5),
-            ("000007", 7),
-            ("000011", 11),
-        )
-        for cantus_id, n in cantus_ids:
-            for _ in range(n):
-                make_fake_chant(cantus_id=cantus_id)
-
-        concordances: dict = update_cached_concordances.get_concordances()
-        with self.subTest(test="Test all Cantus IDs present"):
-            self.assertEqual(len(concordances), len(cantus_ids))
-
-        for cantus_id, n in cantus_ids:
-            concordances_for_id: list = concordances[cantus_id]
-            with self.subTest(n=n):
-                self.assertEqual(len(concordances_for_id), n)
-
     def test_published_vs_unpublished(self):
         published_source: Source = make_fake_source(published=True)
         published_chant: Chant = make_fake_chant(
             source=published_source,
-            cantus_id="123456",
-            incipit="chant in a published source",
+            manuscript_full_text_std_spelling="chant in a published source",
         )
         unpublished_source: Source = make_fake_source(published=False)
         unpublished_chant: Chant = make_fake_chant(
             source=unpublished_source,
-            cantus_id="123456",
-            incipit="chant in an unpublished source",
+            manuscript_full_text_std_spelling="chant in an unpublished source",
         )
 
-        concordances: dict = update_cached_concordances.get_concordances()
-        concordances_for_single_id: list = concordances["123456"]
+        concordances: list = update_cached_concordances.get_concordances()
         self.assertEqual(len(concordances), 1)
 
-        single_concordance: dict = concordances_for_single_id[0]
-        expected_incipit: str = published_chant.incipit
+        single_concordance: dict = concordances[0]
+        # since the fulltext, specified above, contains <= 5 words,
```

Comment:
I would remove this comment and either adjust the test fully in anticipation of #1318 (remove reference to incipit altogether) or keep the creation of fake chants with incipits for this PR and adjust in #1318. The comment is not accurate as of this PR (incipits on fake chants are limited by characters, not words) and in the event that the incipit length is changed in the signal that will soon be introduced, this test could begin to fail.

Reply:
ah - when I was writing this, I was under the impression that our dictionary contained only an `"incipit"` key and nothing relating to fulltext. But I see we do have a `"full_text"` key after all. Thanks for prompting me to think a bit more about this - I'll update it!